tests/test_at.py

```diff
@@ -0,0 +1,174 @@
+from __future__ import annotations
+
+from contextlib import contextmanager, suppress
+
+import numpy as np
+import pytest
+from array_api_compat import (
+    array_namespace,
+    is_cupy_array,
+    is_dask_array,
+    is_jax_array,
+    is_pydata_sparse_array,
+    is_torch_array,
+    is_torch_namespace,
+    is_writeable_array,
+)
+
+from array_api_extra import at
+
+from ._helpers import all_libraries, import_
```

Comment:
WIP still need to refactor this.

@lucascolley this test module necessarily needs to add all libraries (numpy, jax, dask, torch, sparse, cupy) to CI in order to test the functionality, whereas the rest of the test suite exclusively tests vs. array-api-strict.
Should I add these libraries to ci.yml or should I leave it alone and create a brand new github workflow?

Reply:
Seems fine to add to the existing workflow for now. You can create a separate pixi `ci-` env with all the libraries